### ISSUE TITLE
Could not find required component 'xsbti' in 1.5.0 / 0.13.18 build

### ISSUE THREAD

**Issue by sezruby** [sbt/sbt#6447]
## steps
```
git clone https://github.com/microsoft/hyperspace.git
0-1) git fetch origin 9b103c8f
0-2) git checkout origin/9b103c8f
wget https://github.com/sbt/sbt/releases/download/v1.5.0/sbt-1.5.0.tgz
tar -xzf sbt-1.5.0.tgz
sudo apt-get install openjdk-8-jre
path/to/sbt/bin/sbt clean
path/to/sbt/bin/sbt update
path/to/sbt/bin/sbt compile
```

## problem

"sbt compile" failed with the following error: 
```
[info] Loading project definition from /path/to/repo/hyperspace/project
[info] Set current project to hyperspace-core (in build file:/path/to/repo/hyperspace/)
[info] Executing in batch mode. For better performance use sbt's shell
[info] scalastyle using config /path/to/repo/hyperspace/scalastyle-config.xml
[info] scalastyle Processed 76 file(s)
[info] scalastyle Found 0 errors
[info] scalastyle Found 0 warnings
[info] scalastyle Found 0 infos
[info] scalastyle Finished in 40 ms
[success] created output: /path/to/repo/hyperspace/target
[info] Compiling 77 Scala sources to /path/to/repo/hyperspace/target/scala-2.12/classes...
sbt.InvalidComponent: Could not find required component 'xsbti'
        at sbt.ComponentManager.invalid(ComponentManager.scala:57)
        at sbt.ComponentManager.sbt$ComponentManager$$notFound$1(ComponentManager.scala:34)
        at sbt.ComponentManager.sbt$ComponentManager$$createAndCache$1(ComponentManager.scala:37)
        at sbt.ComponentManager$$anonfun$sbt$ComponentManager$$fromGlobal$1$1.apply(ComponentManager.scala:27)
        at sbt.ComponentManager$$anonfun$sbt$ComponentManager$$fromGlobal$1$1.apply(ComponentManager.scala:26)
        at sbt.ComponentManager$$anon$1.call(ComponentManager.scala:50)
        at xsbt.boot.Locks$GlobalLock.withLock(Locks.scala:55)
        at xsbt.boot.Locks$.apply0(Locks.scala:47)
        at xsbt.boot.Locks$.apply(Locks.scala:36)
        at sbt.ComponentManager.lock(ComponentManager.scala:50)
        at sbt.ComponentManager.lockGlobalCache(ComponentManager.scala:49)
        at sbt.ComponentManager.sbt$ComponentManager$$fromGlobal$1(ComponentManager.scala:25)
        at sbt.ComponentManager$$anonfun$files$1$$anonfun$apply$2.apply(ComponentManager.scala:44)
        at sbt.ComponentManager$$anonfun$files$1$$anonfun$apply$2.apply(ComponentManager.scala:44)
        at sbt.ComponentManager.sbt$ComponentManager$$getOrElse$1(ComponentManager.scala:32)
        at sbt.ComponentManager$$anonfun$files$1.apply(ComponentManager.scala:44)
        at sbt.ComponentManager$$anonfun$files$1.apply(ComponentManager.scala:44)
        at sbt.ComponentManager$$anon$1.call(ComponentManager.scala:50)
        at xsbt.boot.Locks$GlobalLock.withLock(Locks.scala:55)
        at xsbt.boot.Locks$.apply0(Locks.scala:47)
        at xsbt.boot.Locks$.apply(Locks.scala:36)
        at sbt.ComponentManager.lock(ComponentManager.scala:50)
        at sbt.ComponentManager.lockLocalCache(ComponentManager.scala:47)
        at sbt.ComponentManager.files(ComponentManager.scala:44)
        at sbt.compiler.IvyComponentCompiler$$anonfun$sbt$compiler$IvyComponentCompiler$$compileAndInstall$1.apply(ComponentCompiler.scala:114)
        at sbt.compiler.IvyComponentCompiler$$anonfun$sbt$compiler$IvyComponentCompiler$$compileAndInstall$1.apply(ComponentCompiler.scala:111)
        at sbt.IO$.withTemporaryDirectory(IO.scala:358)
        at sbt.compiler.IvyComponentCompiler.sbt$compiler$IvyComponentCompiler$$compileAndInstall(ComponentCompiler.scala:111)
        at sbt.compiler.IvyComponentCompiler$$anonfun$apply$1.apply$mcV$sp(ComponentCompiler.scala:102)
        at sbt.IfMissing$Define.apply(ComponentManager.scala:75)
        at sbt.ComponentManager.sbt$ComponentManager$$createAndCache$1(ComponentManager.scala:39)
        at sbt.ComponentManager$$anonfun$sbt$ComponentManager$$fromGlobal$1$1.apply(ComponentManager.scala:27)
        at sbt.ComponentManager$$anonfun$sbt$ComponentManager$$fromGlobal$1$1.apply(ComponentManager.scala:26)
        at sbt.ComponentManager$$anon$1.call(ComponentManager.scala:50)
        at xsbt.boot.Locks$GlobalLock.withChannel$1(Locks.scala:113)
        at xsbt.boot.Locks$GlobalLock.withChannelRetries$1(Locks.scala:91)
        at xsbt.boot.Locks$GlobalLock.$anonfun$withFileLock$1(Locks.scala:119)
        at xsbt.boot.Using$.withResource(Using.scala:12)
        at xsbt.boot.Using$.apply(Using.scala:9)
        at xsbt.boot.Locks$GlobalLock.withFileLock(Locks.scala:119)
        at xsbt.boot.Locks$GlobalLock.ignoringDeadlockAvoided(Locks.scala:71)
        at xsbt.boot.Locks$GlobalLock.withLock(Locks.scala:59)
        at xsbt.boot.Locks$.apply0(Locks.scala:47)
        at xsbt.boot.Locks$.apply(Locks.scala:36)
        at sbt.ComponentManager.lock(ComponentManager.scala:50)
        at sbt.ComponentManager.lockGlobalCache(ComponentManager.scala:49)
        at sbt.ComponentManager.sbt$ComponentManager$$fromGlobal$1(ComponentManager.scala:25)
        at sbt.ComponentManager$$anonfun$files$1$$anonfun$apply$2.apply(ComponentManager.scala:44)
        at sbt.ComponentManager$$anonfun$files$1$$anonfun$apply$2.apply(ComponentManager.scala:44)
        at sbt.ComponentManager.sbt$ComponentManager$$getOrElse$1(ComponentManager.scala:32)
        at sbt.ComponentManager$$anonfun$files$1.apply(ComponentManager.scala:44)
        at sbt.ComponentManager$$anonfun$files$1.apply(ComponentManager.scala:44)
        at sbt.ComponentManager$$anon$1.call(ComponentManager.scala:50)
        at xsbt.boot.Locks$GlobalLock.withChannel$1(Locks.scala:113)
        at xsbt.boot.Locks$GlobalLock.withChannelRetries$1(Locks.scala:91)
        at xsbt.boot.Locks$GlobalLock.$anonfun$withFileLock$1(Locks.scala:119)
        at xsbt.boot.Using$.withResource(Using.scala:12)
        at xsbt.boot.Using$.apply(Using.scala:9)
        at xsbt.boot.Locks$GlobalLock.withFileLock(Locks.scala:119)
        at xsbt.boot.Locks$GlobalLock.ignoringDeadlockAvoided(Locks.scala:71)
        at xsbt.boot.Locks$GlobalLock.withLock(Locks.scala:59)
        at xsbt.boot.Locks$.apply0(Locks.scala:47)
        at xsbt.boot.Locks$.apply(Locks.scala:36)
        at sbt.ComponentManager.lock(ComponentManager.scala:50)
        at sbt.ComponentManager.lockLocalCache(ComponentManager.scala:47)
        at sbt.ComponentManager.files(ComponentManager.scala:44)
        at sbt.ComponentManager.file(ComponentManager.scala:53)
        at sbt.compiler.IvyComponentCompiler.apply(ComponentCompiler.scala:102)
        at sbt.compiler.ComponentCompiler$$anon$2.apply(ComponentCompiler.scala:35)
        at sbt.compiler.AnalyzingCompiler.loader(AnalyzingCompiler.scala:118)
        at sbt.compiler.AnalyzingCompiler.getInterfaceClass(AnalyzingCompiler.scala:128)
        at sbt.compiler.AnalyzingCompiler.call(AnalyzingCompiler.scala:104)
        at sbt.compiler.AnalyzingCompiler.newCachedCompiler(AnalyzingCompiler.scala:62)
        at sbt.compiler.AnalyzingCompiler.newCachedCompiler(AnalyzingCompiler.scala:57)
        at sbt.compiler.CompilerCache$$anon$2.apply(CompilerCache.scala:47)
        at sbt.compiler.AnalyzingCompiler.compile(AnalyzingCompiler.scala:45)
        at sbt.compiler.MixedAnalyzingCompiler$$anonfun$compileScala$1$1.apply$mcV$sp(MixedAnalyzingCompiler.scala:50)
        at sbt.compiler.MixedAnalyzingCompiler$$anonfun$compileScala$1$1.apply(MixedAnalyzingCompiler.scala:50)
        at sbt.compiler.MixedAnalyzingCompiler$$anonfun$compileScala$1$1.apply(MixedAnalyzingCompiler.scala:50)
        at sbt.compiler.MixedAnalyzingCompiler.timed(MixedAnalyzingCompiler.scala:74)
        at sbt.compiler.MixedAnalyzingCompiler.compileScala$1(MixedAnalyzingCompiler.scala:49)
        at sbt.compiler.MixedAnalyzingCompiler.compile(MixedAnalyzingCompiler.scala:64)
        at sbt.compiler.IC$$anonfun$compileInternal$1.apply(IncrementalCompiler.scala:160)
        at sbt.compiler.IC$$anonfun$compileInternal$1.apply(IncrementalCompiler.scala:160)
        at sbt.inc.IncrementalCompile$$anonfun$doCompile$1.apply(Compile.scala:66)
        at sbt.inc.IncrementalCompile$$anonfun$doCompile$1.apply(Compile.scala:64)
        at sbt.inc.IncrementalCommon.cycle(IncrementalCommon.scala:32)
        at sbt.inc.Incremental$$anonfun$1.apply(Incremental.scala:72)
        at sbt.inc.Incremental$$anonfun$1.apply(Incremental.scala:71)
        at sbt.inc.Incremental$.manageClassfiles(Incremental.scala:99)
        at sbt.inc.Incremental$.compile(Incremental.scala:71)
        at sbt.inc.IncrementalCompile$.apply(Compile.scala:54)
        at sbt.compiler.IC$.compileInternal(IncrementalCompiler.scala:160)
        at sbt.compiler.IC$.incrementalCompile(IncrementalCompiler.scala:138)
        at sbt.Compiler$.compile(Compiler.scala:155)
        at sbt.Compiler$.compile(Compiler.scala:141)
        at sbt.Defaults$.sbt$Defaults$$compileIncrementalTaskImpl(Defaults.scala:927)
        at sbt.Defaults$$anonfun$compileIncrementalTask$1.apply(Defaults.scala:918)
        at sbt.Defaults$$anonfun$compileIncrementalTask$1.apply(Defaults.scala:916)
        at scala.Function1$$anonfun$compose$1.apply(Function1.scala:47)
        at sbt.$tilde$greater$$anonfun$$u2219$1.apply(TypeFunctions.scala:40)
        at sbt.std.Transform$$anon$4.work(System.scala:63)
        at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:228)
        at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:228)
        at sbt.ErrorHandling$.wideConvert(ErrorHandling.scala:17)
        at sbt.Execute.work(Execute.scala:237)
        at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:228)
        at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:228)
        at sbt.ConcurrentRestrictions$$anon$4$$anonfun$1.apply(ConcurrentRestrictions.scala:159)

        at sbt.CompletionService$$anon$2.call(CompletionService.scala:28)
        at java.util.concurrent.FutureTask.run(FutureTask.java:266)
```

## expectation

`sbt compile` should succeed.

## notes

As github Azure pipeline updated sbt version to 1.5.0 recently (5 Apr), build & tests are failing in our PRs.

ref https://github.com/actions/virtual-environments/issues/3146

Workaround: use 1.4.9

**Comment by SethTisue** [comment]
I can reproduce this in an empty directory with sbt 0.13.18 as follows:

```
$ mkdir 0.13
$ cd 0.13
$ mkdir project                              
$ echo 'sbt.version=0.13.18' > project/build.properties
$ echo 'class S' > project/S.scala
$ sbt
[info] Loading project definition from /Users/tisue/tmp/0.13/project
[info] Updating {file:/Users/tisue/tmp/0.13/project/}root-0-13-build...
[info] Resolving org.fusesource.jansi#jansi;1.4 ...
[info] Done updating.
[info] Compiling 1 Scala source to /Users/tisue/tmp/0.13/project/target/scala-2.10/sbt-0.13/classes...
sbt.InvalidComponent: Could not find required component 'xsbti'
	at sbt.ComponentManager.invalid(ComponentManager.scala:57)
```

(I'm using the Homebrew-installed sbt launch script.)

full stack trace:

<details>
<pre>
sbt.InvalidComponent: Could not find required component 'xsbti'
	at sbt.ComponentManager.invalid(ComponentManager.scala:57)
	at sbt.ComponentManager.sbt$ComponentManager$$notFound$1(ComponentManager.scala:34)
	at sbt.ComponentManager.sbt$ComponentManager$$createAndCache$1(ComponentManager.scala:37)
	at sbt.ComponentManager$$anonfun$sbt$ComponentManager$$fromGlobal$1$1.apply(ComponentManager.scala:27)
	at sbt.ComponentManager$$anonfun$sbt$ComponentManager$$fromGlobal$1$1.apply(ComponentManager.scala:26)
	at sbt.ComponentManager$$anon$1.call(ComponentManager.scala:50)
	at xsbt.boot.Locks$GlobalLock.withLock(Locks.scala:55)
	at xsbt.boot.Locks$.apply0(Locks.scala:47)
	at xsbt.boot.Locks$.apply(Locks.scala:36)
	at sbt.ComponentManager.lock(ComponentManager.scala:50)
	at sbt.ComponentManager.lockGlobalCache(ComponentManager.scala:49)
	at sbt.ComponentManager.sbt$ComponentManager$$fromGlobal$1(ComponentManager.scala:25)
	at sbt.ComponentManager$$anonfun$files$1$$anonfun$apply$2.apply(ComponentManager.scala:44)
	at sbt.ComponentManager$$anonfun$files$1$$anonfun$apply$2.apply(ComponentManager.scala:44)
	at sbt.ComponentManager.sbt$ComponentManager$$getOrElse$1(ComponentManager.scala:32)
	at sbt.ComponentManager$$anonfun$files$1.apply(ComponentManager.scala:44)
	at sbt.ComponentManager$$anonfun$files$1.apply(ComponentManager.scala:44)
	at sbt.ComponentManager$$anon$1.call(ComponentManager.scala:50)
	at xsbt.boot.Locks$GlobalLock.withLock(Locks.scala:55)
	at xsbt.boot.Locks$.apply0(Locks.scala:47)
	at xsbt.boot.Locks$.apply(Locks.scala:36)
	at sbt.ComponentManager.lock(ComponentManager.scala:50)
	at sbt.ComponentManager.lockLocalCache(ComponentManager.scala:47)
	at sbt.ComponentManager.files(ComponentManager.scala:44)
	at sbt.compiler.IvyComponentCompiler$$anonfun$sbt$compiler$IvyComponentCompiler$$compileAndInstall$1.apply(ComponentCompiler.scala:114)
	at sbt.compiler.IvyComponentCompiler$$anonfun$sbt$compiler$IvyComponentCompiler$$compileAndInstall$1.apply(ComponentCompiler.scala:111)
	at sbt.IO$.withTemporaryDirectory(IO.scala:358)
	at sbt.compiler.IvyComponentCompiler.sbt$compiler$IvyComponentCompiler$$compileAndInstall(ComponentCompiler.scala:111)
	at sbt.compiler.IvyComponentCompiler$$anonfun$apply$1.apply$mcV$sp(ComponentCompiler.scala:102)
	at sbt.IfMissing$Define.apply(ComponentManager.scala:75)
	at sbt.ComponentManager.sbt$ComponentManager$$createAndCache$1(ComponentManager.scala:39)
	at sbt.ComponentManager$$anonfun$sbt$ComponentManager$$fromGlobal$1$1.apply(ComponentManager.scala:27)
	at sbt.ComponentManager$$anonfun$sbt$ComponentManager$$fromGlobal$1$1.apply(ComponentManager.scala:26)
	at sbt.ComponentManager$$anon$1.call(ComponentManager.scala:50)
	at xsbt.boot.Locks$GlobalLock.withChannel$1(Locks.scala:113)
	at xsbt.boot.Locks$GlobalLock.withChannelRetries$1(Locks.scala:91)
	at xsbt.boot.Locks$GlobalLock.$anonfun$withFileLock$1(Locks.scala:119)
	at xsbt.boot.Using$.withResource(Using.scala:12)
	at xsbt.boot.Using$.apply(Using.scala:9)
	at xsbt.boot.Locks$GlobalLock.withFileLock(Locks.scala:119)
	at xsbt.boot.Locks$GlobalLock.ignoringDeadlockAvoided(Locks.scala:71)
	at xsbt.boot.Locks$GlobalLock.withLock(Locks.scala:59)
	at xsbt.boot.Locks$.apply0(Locks.scala:47)
	at xsbt.boot.Locks$.apply(Locks.scala:36)
	at sbt.ComponentManager.lock(ComponentManager.scala:50)
	at sbt.ComponentManager.lockGlobalCache(ComponentManager.scala:49)
	at sbt.ComponentManager.sbt$ComponentManager$$fromGlobal$1(ComponentManager.scala:25)
	at sbt.ComponentManager$$anonfun$files$1$$anonfun$apply$2.apply(ComponentManager.scala:44)
	at sbt.ComponentManager$$anonfun$files$1$$anonfun$apply$2.apply(ComponentManager.scala:44)
	at sbt.ComponentManager.sbt$ComponentManager$$getOrElse$1(ComponentManager.scala:32)
	at sbt.ComponentManager$$anonfun$files$1.apply(ComponentManager.scala:44)
	at sbt.ComponentManager$$anonfun$files$1.apply(ComponentManager.scala:44)
	at sbt.ComponentManager$$anon$1.call(ComponentManager.scala:50)
	at xsbt.boot.Locks$GlobalLock.withChannel$1(Locks.scala:113)
	at xsbt.boot.Locks$GlobalLock.withChannelRetries$1(Locks.scala:91)
	at xsbt.boot.Locks$GlobalLock.$anonfun$withFileLock$1(Locks.scala:119)
	at xsbt.boot.Using$.withResource(Using.scala:12)
	at xsbt.boot.Using$.apply(Using.scala:9)
	at xsbt.boot.Locks$GlobalLock.withFileLock(Locks.scala:119)
	at xsbt.boot.Locks$GlobalLock.ignoringDeadlockAvoided(Locks.scala:71)
	at xsbt.boot.Locks$GlobalLock.withLock(Locks.scala:59)
	at xsbt.boot.Locks$.apply0(Locks.scala:47)
	at xsbt.boot.Locks$.apply(Locks.scala:36)
	at sbt.ComponentManager.lock(ComponentManager.scala:50)
	at sbt.ComponentManager.lockLocalCache(ComponentManager.scala:47)
	at sbt.ComponentManager.files(ComponentManager.scala:44)
	at sbt.ComponentManager.file(ComponentManager.scala:53)
	at sbt.compiler.IvyComponentCompiler.apply(ComponentCompiler.scala:102)
	at sbt.compiler.ComponentCompiler$$anon$2.apply(ComponentCompiler.scala:35)
	at sbt.compiler.AnalyzingCompiler.loader(AnalyzingCompiler.scala:118)
	at sbt.compiler.AnalyzingCompiler.getInterfaceClass(AnalyzingCompiler.scala:128)
	at sbt.compiler.AnalyzingCompiler.call(AnalyzingCompiler.scala:104)
	at sbt.compiler.AnalyzingCompiler.newCachedCompiler(AnalyzingCompiler.scala:62)
	at sbt.compiler.AnalyzingCompiler.newCachedCompiler(AnalyzingCompiler.scala:57)
	at sbt.compiler.CompilerCache$$anon$2.apply(CompilerCache.scala:47)
	at sbt.compiler.AnalyzingCompiler.compile(AnalyzingCompiler.scala:45)
	at sbt.compiler.MixedAnalyzingCompiler$$anonfun$compileScala$1$1.apply$mcV$sp(MixedAnalyzingCompiler.scala:50)
	at sbt.compiler.MixedAnalyzingCompiler$$anonfun$compileScala$1$1.apply(MixedAnalyzingCompiler.scala:50)
	at sbt.compiler.MixedAnalyzingCompiler$$anonfun$compileScala$1$1.apply(MixedAnalyzingCompiler.scala:50)
	at sbt.compiler.MixedAnalyzingCompiler.timed(MixedAnalyzingCompiler.scala:74)
	at sbt.compiler.MixedAnalyzingCompiler.compileScala$1(MixedAnalyzingCompiler.scala:49)
	at sbt.compiler.MixedAnalyzingCompiler.compile(MixedAnalyzingCompiler.scala:64)
	at sbt.compiler.IC$$anonfun$compileInternal$1.apply(IncrementalCompiler.scala:160)
	at sbt.compiler.IC$$anonfun$compileInternal$1.apply(IncrementalCompiler.scala:160)
	at sbt.inc.IncrementalCompile$$anonfun$doCompile$1.apply(Compile.scala:66)
	at sbt.inc.IncrementalCompile$$anonfun$doCompile$1.apply(Compile.scala:64)
	at sbt.inc.IncrementalCommon.cycle(IncrementalCommon.scala:32)
	at sbt.inc.Incremental$$anonfun$1.apply(Incremental.scala:72)
	at sbt.inc.Incremental$$anonfun$1.apply(Incremental.scala:71)
	at sbt.inc.Incremental$.manageClassfiles(Incremental.scala:99)
	at sbt.inc.Incremental$.compile(Incremental.scala:71)
	at sbt.inc.IncrementalCompile$.apply(Compile.scala:54)
	at sbt.compiler.IC$.compileInternal(IncrementalCompiler.scala:160)
	at sbt.compiler.IC$.incrementalCompile(IncrementalCompiler.scala:138)
	at sbt.Compiler$.compile(Compiler.scala:155)
	at sbt.Compiler$.compile(Compiler.scala:141)
	at sbt.Defaults$.sbt$Defaults$$compileIncrementalTaskImpl(Defaults.scala:927)
	at sbt.Defaults$$anonfun$compileIncrementalTask$1.apply(Defaults.scala:918)
	at sbt.Defaults$$anonfun$compileIncrementalTask$1.apply(Defaults.scala:916)
	at scala.Function1$$anonfun$compose$1.apply(Function1.scala:47)
	at sbt.$tilde$greater$$anonfun$$u2219$1.apply(TypeFunctions.scala:40)
	at sbt.std.Transform$$anon$4.work(System.scala:63)
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:228)
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:228)
	at sbt.ErrorHandling$.wideConvert(ErrorHandling.scala:17)
	at sbt.Execute.work(Execute.scala:237)
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:228)
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:228)
	at sbt.ConcurrentRestrictions$$anon$4$$anonfun$1.apply(ConcurrentRestrictions.scala:159)
	at sbt.CompletionService$$anon$2.call(CompletionService.scala:28)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
[error] (compile:compileIncremental) sbt.InvalidComponent: Could not find required component 'xsbti'
Project loading failed: (r)etry, (q)uit, (l)ast, or (i)gnore? 
</pre>
</details>

**Comment by eed3si9n** [comment]
Works on my machine:

```
$ env COURSIER_CACHE=/tmp/cache0 sbt -Dsbt.global.base=/tmp/global -Dsbt.boot.directory=/tmp/boot
[info] [launcher] getting org.scala-sbt sbt 0.13.18  (this may take some time)...
[info] [launcher] getting Scala 2.10.7 (for sbt)...
[info] Loading project definition from /private/tmp/0.13/project
[info] Updating {file:/private/tmp/0.13/project/}root-0-13-build...
[info] Resolving org.fusesource.jansi#jansi;1.4 ...
[info] Done updating.
[info] Set current project to root-0-13 (in build file:/private/tmp/0.13/)
> sbtVersion
[info] 0.13.18
> exit
```

**Comment by SethTisue** [comment]
your suggested command line doesn't fix it on my machine (`env COURSIER_CACHE=/tmp/cache0 sbt -Dsbt.global.base=/tmp/global -Dsbt.boot.directory=/tmp/boot`)

moving my `~/.sbt` aside doesn't fix it either

1.5.0 is not affected, `project/build.properties` must say 0.13.18 or I don't experience this

```
% java -version
openjdk version "1.8.0_282"
OpenJDK Runtime Environment (AdoptOpenJDK)(build 1.8.0_282-b08)
OpenJDK 64-Bit Server VM (AdoptOpenJDK)(build 25.282-b08, mixed mode)
```

if I throw `-debug` in there, the full log is https://gist.github.com/SethTisue/c07446ab9c59771a4e07eda8856e7763

not sure what else to try

**Comment by eed3si9n** [comment]
Sorry I missed one step from your repro:

```
$ echo 'class S' > project/S.scala
```

Forcing the compilation does blow up for me too.

**Comment by eed3si9n** [comment]
I think the workaround would be to give it a new boot directory and use Ivy launcher:

```bash
sbt -Dsbt.boot.directory=/tmp/boot1 -Dsbt.launcher.coursier=false
```

**Comment by eed3si9n** [comment]
We can try to fix this by not using Coursier for sbt 0.13.x.

**Comment by SethTisue** [comment]
The workaround works for me — thanks!!
